BirdClubInfoHub/Mappings/MappingProfile.cs

```diff
@@ -25,9 +24,7 @@ public MappingProfile()
                 .ReverseMap();
 
             CreateMap<FieldTrip, FieldTripDTO>()
-                .ForMember(dest => dest.Status,
```

Comment:
Why remove the conversion ?

Reply:
dto vẫn để status code, ở view cần show ra thì gọi convert